src/app/pages/dashboard/widgets/system/widget-sys-info-active/widget-sys-info-active.component.html

```diff
@@ -150,7 +150,7 @@ <h3>{{ 'System Information' | translate }}</h3>
             <mat-list-item>
               <strong>{{ 'Hostname' | translate }}:</strong>
               @if (isLoaded()) {
-                <span>
+                <span [matTooltip]="systemInfo()?.hostname">
```

Comment:
here we don't need a tooltip since we have a `<strong>{{ 'Hostname' | translate }}:</strong>` which indicates which value we show.